pkgs/applications/audio/exact-audio-copy/default.nix

```diff
@@ -0,0 +1,87 @@
+{ lib
+, stdenv
+, fetchurl
+, makeDesktopItem
+, imagemagick
+, p7zip
+, wine
+, writeShellScriptBin
+, symlinkJoin
+, use64 ? false
+}:
+
+let
+  pname = "exact-audio-copy";
+
+  eac_exe = fetchurl {
+    url = "http://www.exactaudiocopy.de/eac-1.6.exe";
```

Comment:
Is this the same version number as the main package?

Reply:
Yeah, my intention was to keep this in sync as much as possible, only diverging by a Z in X.Y.Z if the package definition itself needs to be updated. Is this reasonable? I'm new to packaging in general and I was having a hard time finding clear conventions.

Reply:
```suggestion
    url = "http://www.exactaudiocopy.de/eac-${lib.versions.majorMinor version}.exe";
```
You mean this?

Reply:
Done, ended up defining a single version up top since this is pretty simple overall